### PR TITLE
Add Source SRV atervinning, SE

### PR DESCRIPTION
**Pull request by bbr111**
Add Source SRV atervinning, SE

### CODE REVIEW

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/srv_se.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            return []
+
+        data = r.json()
+
+        entries = []
+
+        for container in data["results"][0]["containers"]:
+            type=container["contentType"]
+            for calentry in container["calendars"]:
+                dates =  {calentry["startDate"],type}
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/srv_se.py:35]: `dates` is not used

**Review by mampfes — CHANGES_REQUESTED** [review]
Please rename .py and .md files to srvatervinning_se

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/srv_se.py (hunk 1 above)
Comment (bbr111) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/srv_se.py]: changed in last commit

**Comment by mampfes** [comment]
Thanks for your contribution.

**Pull request merged**